web/sites/all/modules/onlinekurslabor/nm_stream/nm_stream.module

```diff
@@ -3685,14 +3685,14 @@ function _nm_stream_filter_mention_process($text, $filter, $format)
 
   // @user
   // @sergio.nachio, @sergio-nachio, @sergio_nachio @sergio_nachio
-  $regex = "/@([A-Z][\w\-]*\s[A-Z][\w\-]*)|@([\w\d]*[\.|\-|_][\w\d]*)/";
+  $regex = "/@([A-Z][\w\-]*\s[A-Z][\w\-]*)|@([\w\d]*[\.|\-|_]?[\w\d]*)/";
 
```

Comment:
sollte das zweite [\w\-] nicht auch [\w\d] sein?
könnt das ganze auch abkürzen und keine zwei ODER-Regex einfügen. Das is aber nich so wichtig.

Reply:
die erste expression sollte @Sergio Nacho-Macho abdecken.
Du meinst bei 2 @Sergio Nacho-Machos im System? Würd sagen ja.. sollten hierfür auch einen einfachen Unit Test schreiben

Reply:
@null https://github.com/n-multimedia/onlinekurslabor/pull/22/files#diff-f136b7e3befd027f545c8ede1b7156cdR66 habe hier die unittest ergänzt.. aus meiner sicht passt das soweit.

Reply:
ah spitze coole sache#!